src/codec_aom.c

```diff
@@ -127,6 +127,7 @@ static avifBool aomCodecGetNextImage(struct avifCodec * codec,
         nextFrame = aom_codec_get_frame(&codec->internal->decoder, &codec->internal->iter);
         if (nextFrame) {
             if (spatialID != AVIF_SPATIAL_ID_UNSET) {
+                // This requires a version of libaom with the fix for https://crbug.com/aomedia/2993.
```

Comment:
I decided to not do a version check here, because I expect the use of the 'lsel' property to be rare.

The version check will need to be a runtime version check because it is not possible to detect this bug fix at compile time. (There are no suitable libaom macros.)